sktime/transformations/series/detrend/tests/test_mstl.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+import pandas as pd
+from sktime.datasets import load_airline
+from sktime.transformations.series.detrend import MSTL
+
+def test_transform_returns_correct_components():
+    # TODO: See if we need to create a fresh time series dataset?
+    # np.random.seed(42)
+    # data = np.random.randn(100)
+    # index = pd.date_range(start="2020-01-01", periods=100, freq="D")
+    # series = pd.Series(data, index=index)
+
+    # Load our default test dataset
+    series = load_airline()
+    series.index = y.index.to_timestamp()
```

Comment:
what is the purpose of this line, where is `y` coming from? I think it remains undefined.